sdk/python/v1beta1/setup.py

```diff
@@ -14,40 +14,44 @@
 
 import setuptools
 
-with open('requirements.txt') as f:
-    REQUIRES = f.readlines()
+REQUIRES = [
+    "certifi>=14.05.14",
+    "six>=1.10",
+    "setuptools>=21.0.0",
+    "urllib3>=1.15.1",
+    "kubernetes>=12.0.0",
```

Comment:
I think that `12.0.0` is so old.
Should we change the required kubernetes client version?

https://github.com/kubernetes-client/python#compatibility-matrix-of-supported-client-versions

Reply:
Sure, let's be in sync with Training Operator SDK.